index.html

```diff
@@ -185,18 +184,6 @@ <h3>New Normative Specifications</h3>
       </p>
     </dd>
 
-    <dt id="webvtt" class="spec">
-      <a href="https://www.w3.org/TR/webvtt1/">WebVTT: The Web Video Text Tracks Format</a>
-    </dt>
-
-    <dd>
-      <p>This specification defines WebVTT, the Web Video Text Tracks format. Its main use is for marking up
-      external text track resources in connection with the HTML &lt;track&gt; element. WebVTT files provide
-      captions or subtitles for video content, and also text video descriptions <a href=
-      "http://www.w3.org/TR/media-accessibility-reqs/">[MAUR]</a>, chapters for content navigation,
-      and more generally any form of metadata that is time-aligned with audio or video content.</p>
-    </dd>
-
```

Comment:
I think we must mention WebVTT _somewhere_ since it has never reached Rec, so it is not clear if it is a "new" or an "existing" technical specification. Do we need a new subsection for "in flight"?

Reply:
how about something like this??

Reply:
Works for me!